assets/prototype/domain/shared/types.d.ts

```diff
@@ -1,9310 +1,9145 @@
-   * calculate the base ticket price from the provided ticket total.
- */
-  reverseCalculate?: Maybe<Scalars['Boolean']>,
-  /** How tickets are displayed in the ui */
-  row?: Maybe<Scalars['Int']>,
-  /** Number of this ticket sold */
-  sold?: Maybe<Scalars['Int']>,
-  /** Start date and time of the Ticket */
-  startDate?: Maybe<Scalars['String']>,
-  /** Number of datetimes this ticket can be used at */
-  uses?: Maybe<Scalars['Int']>,
-  /** Ticket Creator ID */
-  wpUser?: Maybe<Scalars['Int']>,
+	clientMutationId: Scalars['String'];
+	/** Globally unique IDs of the datetimes related to the ticket. Ignored if empty. */
+	datetimes?: Maybe<Array<Maybe<Scalars['ID']>>>;
+	/** Description of Ticket */
+	description?: Maybe<Scalars['String']>;
+	/** End date and time of the Ticket */
+	endDate?: Maybe<Scalars['String']>;
+	/** Flag indicating that this ticket is a default ticket */
+	isDefault?: Maybe<Scalars['Boolean']>;
+	/** Flag indicating whether the ticket is free. */
+	isFree?: Maybe<Scalars['Boolean']>;
+	/** Flag indicating whether this ticket must be purchased with a transaction */
+	isRequired?: Maybe<Scalars['Boolean']>;
+	/** Flag indicating whether there is tax applied on this ticket */
+	isTaxable?: Maybe<Scalars['Boolean']>;
+	/** Maximum quantity of this ticket that can be purchased in one transaction */
+	max?: Maybe<Scalars['Int']>;
+	/** Minimum quantity of this ticket that must be purchased */
+	min?: Maybe<Scalars['Int']>;
+	/** Ticket Name */
+	name?: Maybe<Scalars['String']>;
+	/** The order in which the Datetime is displayed */
+	order?: Maybe<Scalars['Int']>;
+	/** The parent ticket ID */
+	parent?: Maybe<Scalars['ID']>;
+	/** Final calculated price for ticket */
+	price?: Maybe<Scalars['Float']>;
+	/** Globally unique IDs of the prices related to the ticket. Ignored if empty. */
+	prices?: Maybe<Array<Maybe<Scalars['ID']>>>;
+	/** Quantity of this ticket that is available */
+	quantity?: Maybe<Scalars['Int']>;
+	/** Quantity of this ticket that is reserved, but not yet fully purchased */
+	reserved?: Maybe<Scalars['Int']>;
+	/**
+	 * Flag indicating whether ticket calculations should run in reverse and
+	 * calculate the base ticket price from the provided ticket total.
+	 */
+	reverseCalculate?: Maybe<Scalars['Boolean']>;
+	/** How tickets are displayed in the ui */
+	row?: Maybe<Scalars['Int']>;
+	/** Number of this ticket sold */
+	sold?: Maybe<Scalars['Int']>;
+	/** Start date and time of the Ticket */
+	startDate?: Maybe<Scalars['String']>;
+	/** Number of datetimes this ticket can be used at */
+	uses?: Maybe<Scalars['Int']>;
+	/** Ticket Creator ID */
+	wpUser?: Maybe<Scalars['Int']>;
 };
 
 /** The payload for the createEspressoTicket mutation */
 export type CreateEspressoTicketPayload = {
-   __typename?: 'CreateEspressoTicketPayload',
-  clientMutationId: Scalars['String'],
-  espressoTicket?: Maybe<EspressoTicket>,
+	__typename?: 'CreateEspressoTicketPayload';
+	clientMutati
```

Comment:
Fix for:
![14_jan_6](https://user-images.githubusercontent.com/1477453/72355040-62bf2880-36ef-11ea-9d5f-ca36994cf9da.png)